src/content/eduhelx/eduHelxReadMore.mdx

```diff
@@ -0,0 +1,37 @@
+import { IllustrationTextSection } from "../../components/sections/illustrationTextSection.js";
+import { HeadingSection } from "../../components/sections/heading";
+import { MaxWidthWrapper } from "../../components/layout/maxWidthWrapper";
+import photoLectureHall from "../../images/eduhelx/photo-lecture-hall.png";
+import photoWomanWithComputer from "../../images/eduhelx/photo-woman-with-computer.png";
+
+<MaxWidthWrapper padding="1rem 2rem 1rem 2rem" backgroundColor="#FFFFFF" gutterBottom>
```

Comment:
the `padding` here can be removed, since the MaxWidthWrapper doesnt have a `padding` prop